frontend/lib/queries/LoginMutation.ts

```diff
@@ -20,12 +21,23 @@ export interface LoginMutation_login_errors {
   messages: string[];
 }
 
+export interface LoginMutation_login_session {
+  /**
+   * The phone number of the currently logged-in user, or null if not logged-in.
+   */
+  phoneNumber: string | null;
+  /**
+   * The cross-site request forgery (CSRF) token.
+   */
+  csrfToken: string;
+}
+
```

Comment:
I guess this will get repeated through every Apollo-generated TS interface file, which is a bit of a bummer for PR diffs, but it shouldn't affect bundle size since TypeScript's type system is excised when bundles are generated.